### PR TITLE
fix: change story website to non-existant destination

### PR DESCRIPTION
**Pull request by vaeng**
See: https://forum.exercism.org/t/ad-link-in-lucky-numbers-exercise/8204/2

### CODE REVIEW

**Review by neenjaw — APPROVED** [review]

**Comment by neenjaw** [comment]
@vaeng I think you'll need @iHiD or @ErikSchierboom to approve this one.  Not sure how available the core js maintainers are at the moment

File: exercises/concept/lucky-numbers/.docs/instructions.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Instructions
 
 Your friend Kojo is a big fan of numbers.
-He has a small website for playing around with numbers: www.fun-with-numbers.com.
+He has a small website for playing around with numbers: www.kojo-presents-fun-with-numbers.com.
```
Comment (kotp) [inline exercises/concept/lucky-numbers/.docs/instructions.md:4]: Why would we have a link that goes nowhere?  It is not guaranteed to go nowhere and may need to be changed later.

**Review by kotp — CHANGES_REQUESTED** [review]
We could use the established imaginary (yet real) provided example standard, such as `https://fun-with-numbers.example.com` instead.

**Comment by vaeng** [comment]
Maybe it is best to just drop the url. it does not add to the exercise at all.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**